From 1b3ff7cf4e034a46433d2fb589482274f82fd432 Mon Sep 17 00:00:00 2001
From: Hareesh R <hareeshr@broadcom.com>
Date: Thu, 17 May 2012 18:26:49 +0530
Subject: hybrid-nae : Addes shm-id app param

Added shm-id app param for xen hybrid application
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/boot/dts/xlp8xx-hybrid-nae-netos.dts b/arch/mips/boot/dts/xlp8xx-hybrid-nae-netos.dts
index 178d991..b91fabd 100644
--- a/arch/mips/boot/dts/xlp8xx-hybrid-nae-netos.dts
+++ b/arch/mips/boot/dts/xlp8xx-hybrid-nae-netos.dts
@@ -111,6 +111,10 @@
 				vfbtbl-hw-nentries = <20 20 20 20>;
 				dummy-pktdata-addr = <0x0 0x0ff00000>; // 255 MB
 			};
+
+			app-param {
+				shm-id = <0>;
+			};
 		};
 	};
 
-- 
1.7.1

